### ISSUE TITLE
Oculus Browser Support (currently causing force crash of web page)

### ISSUE THREAD

**Issue by jumpjosh** [mistic100/Photo-Sphere-Viewer#454]
Hello!

Opening a web page with PSV on it causes an instant crash in the Oculus browser (Quest).
Does anyone know why this happens and if there's a fix?

You can try it on the PSV home page, it's possible to visit the plugin page because there is no player there, but if you go to the home page, the oculus browser crashes.

![image](https://user-images.githubusercontent.com/73508680/97269924-65f06300-1882-11eb-9a64-115485f5791c.png)
[PSV-crash.zip](https://github.com/mistic100/Photo-Sphere-Viewer/files/5443643/PSV-crash.zip)

I've also attached a Zip with a video showing the crash.

The Oculus browser is apparently based on Chromium: https://developer.oculus.com/documentation/oculus-browser/

Thanks for any help, and thank you so much for the amazing player. I've tried so many over the years and this is my favourite. I'm learning JavaScript at the moment and will hopefully be able to contribute at some point in the future.

Josh

**Comment by mistic100** [comment]
Is it working on a standalone page ?

The home page displays the viewer in a iframe from jsfiddle, perhaps this is what causes the crash.

**Comment by jumpjosh** [comment]
Unfortunately not on a standalone page either.

I tried outside of jsfiddle with both an iFrame and with it just in a div, both had the same crash. I also tested the omnivirt player which sits in an iframe and it works, so that signals it may not be iframe related.

I just tested on the firefox reality browser app on oculus quest, and it works there in the jsfiddle embed and on standalone pages.

So something that causes a browser crash on oculus browser but not firefox. Unfortunately, I expect most people will be using oculus browser as it's pretty baked-in to the UI.

Is there anything I can do to help investigate?
